src/index.js

```diff
@@ -174,25 +190,22 @@ function getValuesAsList(value) {
      // join items which are within parenthese
      // luckily `calc (100% - 5px)` is invalid syntax and it must be `calc(100% - 5px)`, otherwise this would be even more complex
     .reduce(({list, state}, item) => {
-      if (includes(item, '(')) {
-        state.withinParens = true
-        list.push(item)
-      } else if (state.withinParens) {
-        if (includes(item, ')')) {
-          state.withinParens = false
-        }
+      const openParansCount = (item.match(/\(/g) || []).length
```

Comment:
Doing this actually reduced the cyclomatic complexity which is cool. Thanks @xymostech :)